scripts/Install

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Program information
+prog_name="BorgBackup"
+version="1.1.10"
```

Comment:
version number should be a commandline parameter.

nobody wants to update hardcoded version numbers each time a new release is made.